r/tests/testthat/test-dataset.R

```diff
@@ -98,6 +98,16 @@ test_that("Simple interface for datasets", {
   )
 })
 
+test_that("dim method returns the correct number of rows and columns",{
+  ds <- open_dataset(dataset_dir, partitioning = schema(part = uint8()))
+
+  expect_equivalent(
+    dim(collect(ds)),
```

Comment:
IMO this assertion would read more cleanly if you gave the exact numbers, i.e. `expect_identical(dim(ds), c(20L, 8L))` or whatever the numbers are.

Reply:
Great point.